Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/App.js	(date 1515699191000)
+++ src/App.js	(revision )
@@ -18,6 +18,7 @@
 
 
 
+
 const customHistory = createBrowserHistory();
 
 class App extends Component {
@@ -27,6 +28,7 @@
                 <div>
                     <Route exact path="/" component={Home} />
                     <Route path="/blog/" component={Blog} />
+
                     <Route path="/pages/" component={Pages} />
                     <Route path="/megaMenu/" component={MegaMenu} />
                     <Route path="/eccomerse/" component={Eccomerse} />
